fe2-android/app/src/main/res/layout/fragment_manage_election.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/fragment_manage_election"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <include
+            android:id="@+id/manage_election_tab_back"
+            layout="@layout/tab_back"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/manage_election_guideline_horizontal"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_percent="@dimen/guideline_horizontal_tab" />
+
+
+        <ScrollView
+            android:id="@+id/manage_election_scrollview"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="50dp"
```

Comment:
You have some dimensions left in this file, layout_election_display.xml and layout_event.xml that should be in the dimens.xml file